docs/development/affiliated-packages.rst

```diff
@@ -132,7 +132,8 @@ Starting a new package
     git add <packagename>/conftest.py
 
    You can also uncomment the line ``enable_deprecations_as_exceptions()`` if
-   you want deprecation warnings to make tests fail.
+   you want deprecation warnings to make tests fail. There are also
+   options to customize the information to be printed when running the tests.
```

Comment:
@bsipocz - would be good to actually say that in this PR, though.  Something like:  "There are also options to customize the information to be printed when running the tests. The package template has comments in the conftest.py file that indicate what they are."